src/languages/es.js

```diff
@@ -280,6 +280,9 @@ export default {
         sayHello: '¡Saluda!',
         usePlusButton: '\n\n¡También puedes usar el botón + de abajo para enviar o pedir dinero!',
     },
+    mentionSuggestions: {
+        hereAlternateText: 'Notificar a todos en línea en esta sala',
```

Comment:
Got that from deepl.com, no idea if it's correct

Reply:
Asked for a review: https://expensify.slack.com/archives/C21FRDWCV/p1684165236405219

Reply:
Let's use: 

> Notificar a todos los que estén en linea de esta sala